### DISCUSSION TITLE
Spyder 5 - connection problem

### DISCUSSION THREAD

**Discussion by roeek37** [erdewit/ib_insync#455] [Q&A]
I'm looking to work with IB insync on spyder. 
what i'm using is:

 import nest_asyncio
 nest_asyncio.apply()
from ib_insync import *
#util.startLoop()
ib = IB()
ib.connect('127.0.0.1', 7497, clientId=999)
print('connected')

I have tried to uncomment both nest_asyncio part and util.startLoop().
It seems to solve the "loop is already running" error,
but nothing happen after that (keeps running) and there is no connection 
to the TWS.

I'm using:
- python 3.9
- spyder 5
- Win 10

I don't understand what I am doing wrong, any suggestions ?

**Comment by yiorgosn** [comment]
I am not a programmer so I will explain in the best way I can.  When I 
updated to Spyder 5 I also lost the functionality of nest_asyncio and 
couldn't connect to AWS so I reached out to the Spyder team who fixed it 
by tweaking and updating the spyder_kernels package.  Make sure you have 
installed the latest version of the spyder_kernels package.

Good luck!


------ Original Message ------
From: "roeek37" ***@***.***>
To: "erdewit/ib_insync" ***@***.***>
Cc: "Subscribed" ***@***.***>
Sent: 3/21/2022 7:19:11 PM
Subject: [erdewit/ib_insync] Spyder 5 - connection problem (Discussion 
#455)
